packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json

```diff
@@ -68,6 +69,8 @@
   "ssi_information_request_page_form_tax_registration_number_label": "RSIN",
   "ssi_information_request_page_form_contact_first_name_label": "Contact voornaam",
   "ssi_information_request_page_form_contact_last_name_label": "Contact achternaam",
+  "ssi_information_request_page_form_contact_bsn_label": "Contact BSN",
+  "ssi_information_request_page_form_tax_registration_code_label": "Belastingregistratiecode",
```

Comment:
we already had localization for these fields right?

Reply:
"ssi_information_request_page_form_tax_registration_number_label"

Reply:
that's `RSIN` and not the full name. based on the design it should be the fullname

Reply:
Check with Maarten, because then we should change it on all places, meaning we still have duplicates

Reply:
following our discussion, changed it